csatar-plugins/csatar/models/scout/columns.yaml

```diff
@@ -43,3 +43,40 @@ columns:
         type: number
         searchable: true
         sortable: true
+    is_active:
+        label: is_active
+        type: switch
+        searchable: true
+        sortable: true
+    chronic_illnesses:
+        label: chronic_illnesses
+        type: text
+        searchable: true
+        sortable: true
+    allergies:
+        label: chronic_illnesses
+        type: text
+        searchable: true
+        sortable: true
+    legal_relationship_id:
+        label: legal_relationship_id
+        type: text
+        searchable: true
+        sortable: true
+        relation: legal_relationship
+        valueFrom: title
+    special_diet_id:
```

Comment:
If we want to display values from these fields in the list, all these columns should have `relation` and `valueFrom` properties specified.